docs/mddocs/Quickstart/llamacpp_portable_zip_gpu_quickstart.md

```diff
@@ -0,0 +1,314 @@
+# Run Llama.cpp Portable Zip on Intel GPU with IPEX-LLM
+<p>
+  <b>< English</b> | <a href='./llamacpp_portable_zip_gpu_quickstart.zh-CN.md'>中文</a> >
+</p>
+
+This guide demonstrates how to use [llama.cpp portable zip](https://github.com/intel/ipex-llm/releases/tag/v2.2.0-nightly) to directly run llama.cpp on Intel GPU with `ipex-llm` (without the need of manual installations).
+
+> [!NOTE]
+> Llama.cpp portable zip has been verified on:
+> - Intel Core Ultra processors
+> - Intel Core 11th - 14th gen processors
+> - Intel Arc A-Series GPU
+> - Intel Arc B-Series GPU
+> Currently, IPEX-LLM only provides llama.cpp portable zip on Windows. 
+
+## Table of Contents
+- [Windows Quickstart](#windows-quickstart)
+  - [Prerequisites](#prerequisites)
+  - [Step 1: Download and Unzip](#step-1-download-and-unzip)
+  - [Step 3: Runtime Configuration](#step-2-runtime-configuration)
+  - [Step 3: Run GGUF models](#step-3-run-gguf-models)
+- [Tips & Troubleshooting](#tips--troubleshooting)
+  - [Error: Detected different sycl devices](#error-detected-different-sycl-devices)
+  - [Multi-GPUs usage](#multi-gpus-usage)
+  - [Performance Environment](#performance-environment)
+
+## Windows Quickstart
+
+### Prerequisites
+
+Check your GPU driver version, and update it if needed:
+
+- For Intel Core Ultra processors (Series 2) or Intel Arc B-Series GPU, we recommend updating your GPU driver to the [latest](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html)
+
+- For other Intel iGPU/dGPU, we recommend using GPU driver version [32.0.101.6078](https://www.intel.com/content/www/us/en/download/785597/834050/intel-arc-iris-xe-graphics-windows.html)
+
+### Step 1: Download and Unzip
+
+Download IPEX-LLM Llama.cpp portable zip for Windows users from the [link](https://github.com/intel/ipex-llm/releases/tag/v2.2.0-nightly).
+
+Then, extract the zip file to a folder.
+
+### Step 2: Runtime Configuration
+
+- Open "Command Prompt" (cmd), and enter the extracted folder through `cd /d PATH\TO\EXTRACTED\FOLDER`
+- To use GPU acceleration, several environment variables are required or recommended before running `llama.cpp`.
+```cmd
+set SYCL_CACHE_PERSISTENT=1
+```
+- For multi-GPUs user, go to [Tips](#select-specific-gpu-to-run-llamacpp-when-multiple-ones-are-available) for how to select specific GPU.
```

Comment:
There seems to be no section called #select-specific-gpu-to-run-llamacpp-when-multiple-ones-are-available?